scootapi/client/smoke_test_cmd.go

```diff
@@ -61,6 +61,41 @@ func (c *Client) runSmokeTest(cmd *cobra.Command, args []string) error {
 		}()
 	}
 
+	// current status of every job
+	jobStatusMap := make(map[string]scoot.Status)
```

Comment:
I don't think jobStatusMap is threadsafe, it's being accessed from two different goroutines.  And it seems like we don't need to track anything in this scope, we can have: a goroutine which keeps this map, receives updates from generateAndRunJob() and  ranges over ticker to print updates. And I'd create a type JobAndStatus to send over the ch rather than a map with a single entry.